### ISSUE TITLE
Apple Ecosystem Support Enhancement Ideas

### ISSUE THREAD

**Issue by doomguy** [simplex-chat/simplex-chat#2326]
Hi there,

just some ideas on how to improve the support of your app(s) for the Apple ecosystem:

- homebrew support (brew install simplex-chat-cli)
- signing your macho binary with your developer certificate
- build universal binaries (arm64+x86_64)
  - https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary
- compile the iOS App with support for macOS (so Apple Silicon users can just use the app from the Store)
  - https://developer.apple.com/documentation/apple-silicon/running-your-ios-apps-in-macos

Thanks for considering!
/Doomguy

**Comment by jooray** [comment]
I would add at least a separate arm64 library. The x86_64 does not work through emulation because of missing x86_64 libs (if you have arm64 homebrew, it has arm64 openssl@1.1).

**Comment by epoberezkin** [comment]
have you seen desktop beta for Mac?

**Comment by jooray** [comment]
Yes, I use the desktop beta, but I would like to use command-line version from scripts.

**Comment by epoberezkin** [comment]
desktop beta is now signed and notarised btw. 

We won't be signing CLI - you can compile it from source if needed.

**Comment by jooray** [comment]
It does not currently compile on macos arm64 with toolchain in the os/brew though :(.

https://github.com/simplex-chat/simplex-chat/issues/3000

**Comment by jooray** [comment]
Adding simplex-chat cli to homebrew would be very useful
